keras_cv/utils/preprocessing.py

```diff
@@ -14,6 +14,8 @@
 import tensorflow as tf
 from tensorflow.keras import backend
 
+import keras_cv.bounding_box.converters as bbox_formater
```

Comment:
even though we are removing this, remember to fully expand names:

bbox -> bounding_box